rules/integrations/aws/discovery_new_terms_sts_getcalleridentity.toml

```
@@ -93,10 +95,23 @@ timestamp_override = "event.ingested"
 type = "new_terms"
 
 query = '''
-event.dataset: "aws.cloudtrail" and event.provider: "sts.amazonaws.com" and event.action: "GetCallerIdentity"
-and not aws.cloudtrail.user_identity.type: "AssumedRole"
+event.dataset: "aws.cloudtrail"
+    and event.provider: "sts.amazonaws.com"
+    and event.action: "GetCallerIdentity"
+    and event.outcome: "success"
+    and not aws.cloudtrail.user_identity.type: "AssumedRole"
 '''
 
+[rule.investigation_fields]
+field_names = [
+    "event.action",
+    "event.outcome",
+    "cloud.region",
+    "user_agent.original",
+    "user.name",
+    "aws.cloudtrail.user_identity.type",
+    "aws.cloudtrail.user_identity.arn"
```

```suggestion
    "user.name",
    "aws.cloudtrail.user_identity.arn",
    "aws.cloudtrail.user_identity.type",
    "event.action",
    "cloud.region",
    "user_agent.original"
```
event.outcome is already part of query and always successful, moved the WHO fields to the top

---

See https://github.com/elastic/detection-rules/pull/4261#discussion_r1834863713 and https://github.com/elastic/detection-rules/pull/4261#discussion_r1834866860